### PR TITLE
fix(filter components): ui fixes

### PR DESCRIPTION
**Pull request by anitavincent**
#### What is the purpose of this pull request?
Fix some UI and UX issues.

#### How should this be manually tested?
Check all filter stories

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Quality improvement (tests or refactors)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Trivial change (small fix or feature that doesn't impact functionalities)
- [ ] Requires change to documentation, which has been updated accordingly

### CODE REVIEW

File: packages/admin-ui/src/filters/__stories__/filters-searchable.stories.tsx [hunk 1]
```diff
@@ -122,10 +123,8 @@ export function Async() {
   useEffect(() => {
     if (status === 'loading') return
 
-    combobox.value && !combobox.matches.length
-      ? setStatus('not-found')
-      : setStatus('ready')
-  }, [combobox.matches])
+    searchValue && !matches.length ? setStatus('not-found') : setStatus('ready')
```
Comment (Lenderson) [inline packages/admin-ui/src/filters/__stories__/filters-searchable.stories.tsx:126]: Maybe here would fit better an `if statement` or use the ternary expression with the status value. 
```suggestion
    const newStatus = searchValue && !matches.length ? 'not-found' : 'ready'
   setStatus(newStatus)
```

File: packages/admin-ui/src/filters/filter-group.state.tsx [hunk 2]
```diff
@@ -1,15 +1,17 @@
 import type { GenericFilterStateReturn } from './filter/filter.state'
 
 export function useFilterGroupState(props: UseFilterGroupStateReturn) {
-  const { filterStates } = props
+  const { filterStates, onClear: onClearCb = () => {} } = props
 
   const onClear = () => {
     filterStates.forEach((state) => state.onClear())
+    onClearCb()
```
Comment (Lenderson) [inline packages/admin-ui/src/filters/filter-group.state.tsx:8]: As the callback function is optional, you could use the optional chaining operator and remove the default value of the cb function:
```suggestion
    onClearCb?.()
```

File: packages/admin-ui/src/filters/filter-status.tsx [hunk 3]
```diff
@@ -56,7 +56,12 @@ export function FilterStatus(props: FilterStatusProps) {
       {status === 'error' && (
         <Box csx={style.statusLayout} className="__admin-ui-filter-status">
```
Comment (lucasaarcoverde) [inline packages/admin-ui/src/filters/filter-status.tsx:57]:
```suggestion
        <Stack csx={style.statusLayout} className="__admin-ui-filter-status">
```

File: packages/admin-ui/src/filters/filter-status.tsx [hunk 4]
```diff
@@ -56,7 +56,12 @@ export function FilterStatus(props: FilterStatusProps) {
...
           <Text variant="title2">{formatMessage('errorTitle')}</Text>
-          <Button variant="tertiary" onClick={onRetry} bleedX>
+          <Button
+            variant="tertiary"
+            onClick={onRetry}
+            csx={{ marginTop: '$m' }}
```
Comment (lucasaarcoverde) [inline packages/admin-ui/src/filters/filter-status.tsx:62]:
```suggestion
```

File: packages/admin-ui/src/filters/filter-status.tsx [hunk 5]
```diff
@@ -43,23 +43,22 @@ export function FilterStatus(props: FilterStatusProps) {
             direction="row"
             csx={{
               width: '100%',
-              '> *': { height: 32, width: 62 },
```
Comment (anitavincent) [inline packages/admin-ui/src/filters/filter-status.tsx:46]: This wasn't working. The children weren't rendered at all

**Review by lucasaarcoverde — APPROVED** [review]

**Review by Lenderson — APPROVED** [review]

**Pull request merged**